src/sql/src/session/vars.rs

```diff
@@ -1103,6 +1103,50 @@ const SSH_KEEPALIVES_IDLE: ServerVar<Duration> = ServerVar {
     internal: true,
 };
 
+/// Enables `socket.keepalive.enable` for rdkafka client connections. Defaults to true.
+const KAFKA_SOCKET_KEEPALIVE: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("kafka_socket_keepalive"),
+    value: &mz_kafka_util::client::DEFAULT_KEEPALIVE,
+    description:
+        "Enables `socket.keepalive.enable` for rdkafka client connections. Defaults to true.",
+    internal: true,
+};
+
+/// Controls `socket.timeout.ms` for rdkafka client connections. Defaults to the rdkafka default.
+const KAFKA_SOCKET_TIMEOUT: ServerVar<Duration> = ServerVar {
+    name: UncasedStr::new("kafka_socket_timeout"),
+    value: &mz_kafka_util::client::DEFAULT_SOCKET_TIMEOUT,
+    description: "Controls `socket.timeout.ms` for rdkafka \
+        client connections. Defaults to the rdkafka default.",
+    internal: true,
+};
+
+/// Controls `transaction.timeout.ms` for rdkafka client connections. Defaults to the rdkafka default.
+const KAFKA_TRANSACTION_TIMEOUT: ServerVar<Duration> = ServerVar {
+    name: UncasedStr::new("kafka_transaction_timeout"),
+    value: &mz_kafka_util::client::DEFAULT_TRANSACTION_TIMEOUT,
+    description: "Controls `transaction.timeout.ms` for rdkafka \
+        client connections. Defaults to the rdkafka default.",
```

Comment:
Should we mention the constraints here to maximize setting the correct value whenever anyone fiddles with them?

Reply:
sure!